app/api/v1/router.py

```diff
@@ -20,3 +21,4 @@
     prefix="/users/{user_id}/comprehensive_evaluations",
     tags=["comprehensive_evaluations"],
 )
+router.include_router(memo.router, prefix="/memos", tags=["memos"])
```

Comment:
@GideokKim 
메모도 상위 테이블 계층이 있다면 url 에도 표현될 수 있게 아래 처럼 prefix로 붙여주면 어떨까용
```suggestion
router.include_router(memo.router, prefix="/users/{user_id}/catetories/{category_id}/memos", tags=["memos"])
```

Reply:
반영하였습니당